### PR TITLE
final submission

### PR DESCRIPTION
**Pull request by ssoemoe**

### CODE REVIEW

File: assets/testing_instructions.MD [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+1) download the google assistant or Alexa app on your phone/tablet
+2) login with the email and password provided below
+
+username: cronpatientemail@gmail.com
+password: 50f407f21003
+
+3) in the main/home tab:
+     tap on the microphone for google 
+     tap on alexa icon for alexa
+4) say "hey google talk to clinic attendant" or "alexa open clinic attendant"
```
Comment (sinapples) [inline assets/testing_instructions.MD]: Say "Hey Google, talk to Clinic Attendant" or "Alexa, open Clinic Attendant"

File: assets/testing_instructions.MD [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Testing Instructions
+
+1) download the google assistant or Alexa app on your phone/tablet
+2) login with the email and password provided below
+
+username: cronpatientemail@gmail.com
+password: 50f407f21003
+
+3) in the main/home tab:
+     tap on the microphone for google
```
Comment (sinapples) [inline assets/testing_instructions.MD]: add capitals for Alexa and Google and the Start of every line

File: assets/testing_instructions.MD (hunk 1 above)
Comment (ssoemoe) [reply assets/testing_instructions.MD]: that's all ,right?

File: assets/testing_instructions.MD [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+1) Download the Google Assistant or Alexa app on your phone/tablet
+2) Login with the email and password provided below
+
+username: cronpatientemail@gmail.com
+password: 50f407f21003
+
+3) In the main/home tab:
+     Tap on the microphone for Google 
+     Tap on alexa icon for Alexa
```
Comment (sinapples) [inline assets/testing_instructions.MD]: missed one sorry

**Review by sinapples — APPROVED** [review]

**Pull request merged**